cuequivariance_torch/cuequivariance_torch/primitives/tensor_product.py

```diff
@@ -190,7 +240,7 @@ def _tensor_product_fx(
             seg_shape = descriptor.get_segment_shape(-1, path)
             outputs += [
                 out.reshape(
-                    out.shape[: out.ndim - len(seg_shape)] + (math.prod(seg_shape),)
+                    out.shape[: out.ndim - len(seg_shape)] + (prod(seg_shape),)
```

Comment:
I thought you said that you gave up on enabling script for the fx code. What is this for then?

Reply:
Oh it's in that part! Thanks, I will revert it.

Reply:
The ones in _reshape are actually used in FusedTensorProductOp3 - so probably best to keep them all consistent.